nixos/modules/services/networking/wpa_supplicant.nix

```diff
@@ -75,11 +101,48 @@ in {
               '';
             };
 
+            authProtocols = mkOption {
+              # This enables 802.11r by default
+              default = [ "WPA-PSK" "WPA-EAP" "FT-PSK" "FT-EAP" ];
+              # The list can be obtained by running this command
+              # awk '
+              #   /^# key_mgmt: /{ run=1 }
+              #   /^#$/{ run=0 }
+              #   /^# [A-Z0-9-]{2,}/{ if(run){printf("\"%s\"\n", $2)} }
+              # ' /run/current-system/sw/share/doc/wpa_supplicant/wpa_supplicant.conf.example
+              type = types.listOf (types.enum [
+                "WPA-PSK"
```

Comment:
Is WPA2 and WPA3 in here and just named differently?

Reply:
I'm not sure what's you're asking. WPA-Personal is `WPA-PSK` and WPA3-Personal is `SAE`.